curvefs/src/client/warmup/warmup_manager.cpp

```diff
@@ -265,7 +242,25 @@ void WarmupManagerS3Impl::FetchDentry(fuse_ino_t key, fuse_ino_t ino,
         return;
 
     } else if (FsFileType::TYPE_SYM_LINK == dentry.type()) {
-        // skip links
+        std::string symLink;
+        CURVEFS_ERROR statusCode =
+            fuseOpReadLink_(0, dentry.inodeid(), &symLink);
```

Comment:
Maybe nullptr is more readable than 0.

Reply:
> Maybe nullptr is more readable than 0.

Since this is the inodeid, wouldn't nullptr cause confusion: this should be a pointer?

Reply:
> Maybe nullptr is more readable than 0.

fix